### ISSUE TITLE
Pix3D: Parameters for converting .obj to .binvox

### ISSUE THREAD

**Issue by SbnSl** [hzxie/Pix2Vox#25]
Hi, What are the parameters when converting .obj files to .binvox files for Pix3D? I am using the following:
./binvox -d 32 'model.obj'
When I am testing the checkpoint on ShapeNet dataset, I can reproduce the exact results reported in the paper. However, I can not reproduce the results on Pix3D. 

1- I was wondering if any other pre-processing (e.g., rotation) should be applied on the binvox files?  
2- Should I use the same checkpoint (Pix2Vox-A-ShapeNet.pth) when testing on Pix3D?

**Comment by hzxie** [comment]
1. I cannot remember the parameters for `binvox` clearly. After running `binvox`, the object is normalized to the center and has the resolutions of 32. The training data is generated with the [Render For CNN](https://github.com/ShapeNet/RenderForCNN) pipeline.
2. The checkpoint for Pix3D is missing due to the disk damage of servers. You cannot directly use the ShapeNet checkpoint.

**Comment by ChauncyFr** [comment]
> Hi, What are the parameters when converting .obj files to .binvox files for Pix3D? I am using the following:
> ./binvox -d 32 'model.obj'
> When I am testing the checkpoint on ShapeNet dataset, I can reproduce the exact results reported in the paper. However, I can not reproduce the results on Pix3D.
> 
> 1- I was wondering if any other pre-processing (e.g., rotation) should be applied on the binvox files?
> 2- Should I use the same checkpoint (Pix2Vox-A-ShapeNet.pth) when testing on Pix3D?

./binvox -e -d 32 +model.obj path
